build.gradle

```diff
@@ -25,9 +25,10 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.google.android.exoplayer:exoplayer:2.11.7"
+    implementation "com.google.android.exoplayer:exoplayer:2.13.3"
```

Comment:
seems that 2.15.x already exists, have you tried it? @rikner

Reply:
Hey, yes I also tried 2.15.
But it had another weird API change, which I wasn't able to resolve. They changed
`fun onPlayerError(error: ExoPlaybackException)` 
to
`fun onPlayerError(error: PlaybackException)`
and I couldn't figure out how to refactor our implementation of this. Would leave this for later :-P